WTWSMS/decisions/LI_target_decisions.txt

```diff
@@ -1,4 +1,4 @@
-targetted_decisions = {	
+decisions = {	
```

Comment:
Is the player notified when they have a chance to convert to the attacker's religion? If not, it's not proper.